### PR TITLE
Added Reflection

### LINKED ISSUE #57: Hazard Analysis - Reflection

### PR DESCRIPTION
**Pull request by harshc22**
closes #57 
- reflection added
- fixed syntax error

### CODE REVIEW

**Review by KrishPatel27 — APPROVED** [review]
Looks good!

File: docs/HazardAnalysis/HazardAnalysis.tex [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 \hypersetup{ colorlinks=true,       % false: boxed links; true: colored links
     linkcolor=red,          % color of internal links (change box color with
-    linkbordercolor)
+    linkbordercolor=blue,
```
Comment (ShazimR) [inline docs/HazardAnalysis/HazardAnalysis.tex:11]: this is not the correct fix, this should be in the comment as well

**Review by ShazimR — CHANGES_REQUESTED** [review]

**Review by ShazimR — APPROVED** [review]
approving, however someone else should confirm before merging

**Comment by ShazimR** [comment]
> approving, however someone else should confirm before merging

@harshc22 confirmed offline

**Pull request merged**